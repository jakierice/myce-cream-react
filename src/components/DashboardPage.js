import React from 'react';

const ExpenseDashboardPage = () => (
  <div className="content-container">
    <h1>DASHBOARD PAGE CONTENT GOES HERE</h1>
  </div>
);

export default ExpenseDashboardPage;
