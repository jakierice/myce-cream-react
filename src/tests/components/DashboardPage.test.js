import React from 'react';
import { shallow } from 'enzyme';
import DashboardPage from '../../components/DashboardPage';

test('should render DashboadPage correctly', () => {
  const wrapper = shallow(<DashboardPage />);
  expect(wrapper).toMatchSnapshot();
});
