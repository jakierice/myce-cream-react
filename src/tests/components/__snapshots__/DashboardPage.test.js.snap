// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render DashboadPage correctly 1`] = `
<div>
  <h1>
    DASHBOARD PAGE CONTENT GOES HERE
  </h1>
</div>
`;
